connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -106,13 +106,23 @@ export async function githubUpsertIssueActivity(
       page: resultPage,
     });
     resultPageLogger.info("Fetching GitHub issue comments result page.");
-    const comments = await getIssueCommentsPage(
-      installationId,
-      repoName,
-      login,
-      issueNumber,
-      resultPage
-    );
+    let comments = undefined;
+    try {
+      comments = await getIssueCommentsPage(
+        installationId,
+        repoName,
+        login,
+        issueNumber,
+        resultPage
+      );
+    } catch (e: any) {
```

Comment:
Warning should be removed before merging :pray:

Reply:
ack